OpenTelemetry.sln

```diff
@@ -106,12 +106,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ISSUE_TEMPLATE", "ISSUE_TEM
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "workflows", "workflows", "{E69578EB-B456-4062-A645-877CD964528B}"
 	ProjectSection(SolutionItems) = preProject
-		.github\workflows\docs.yml = .github\workflows\docs.yml
+		.github\workflows\docfx.yml = .github\workflows\docfx.yml
```

Comment:
Thanks for the catch :)

Reply:
thank you for the dotnet-format information 👍